nmmo/core/realm.py

```diff
@@ -42,7 +43,7 @@ def __init__(self, config):
     config.MAP_GENERATOR(config).generate_all_maps()
 
     self.datastore = NumpyDatastore()
-    for s in [TileState, EntityState, ItemState]:
+    for s in [TileState, EntityState, ItemState, EventState]:
```

Comment:
Are you sure you don't want to have EventState, AttackEventState, ItemEventState? That way you can have appropriate properties for each one, instead of overriding "number" and having "gold" all events.

Not a big deal, if you prefer this design

Reply:
Yeah.. I'll probably revisit this after actually using/analyzing some events because this design is pushing some mapping work to downstream tasks. I'll keep watching and fix it if problems start to emerge. 

Thank you for your feedback, @daveey!